docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Send Batch Transactions
+sidebar_position: 4
+---
+
+### Batch Transactions
+
+A batch refers to a group of calls inside a single user operation. This can be an approve and send transaction in one call. The executeBatch() method is used.
+
+The executeBatch() method takes in an array of _`Call()`_. With each method taking in 3 keys: to, value and data.
+
+```dart
+executeBatch([
+  Call(to: to, value: value, data: data),
+  Call(to: to, value: value, data: data),
+  Call(to: to, value: value, data: data),
+]);
+```
+
+| Parameter | Type       | Status     | Description                                      |
+| :-------: | ---------- | ---------- | ------------------------------------------------ |
+|  **to**   | _address_  | _required_ | The address where the user want to send Tokens.  |
+| **value** | _number_   | _required_ | The amount of Tokens the user is sending         |
+| **data**  | _Function_ |            | Encoded Function Data for Smart Contract Methods |
+
+Example **`transfer_and_approve.dart`**
+
```

Comment:
The example isn't incorrect. It doesn't make sense to transfer and approve. It needs to be approve and transfer.
Secondly, the code example needs to be updated accordingly. Here's an example:

```
  final res = await fuseSDK.executeBatch(
    [
      // Approve ERC20 Token call
      Call(
        to: tokenAddress,
        value: BigInt.zero,
        data: ContractsUtils.encodeERC20ApproveCall(
          tokenAddress,
          recipientAddress,
          amountInWei,
        ),
      ),
      // Transfer ERC20 Token call
      Call(
        to: tokenAddress,
        value: BigInt.zero,
        data: ContractsUtils.encodeERC20TransferCall(
          tokenAddress,
          recipientAddress,
          amountInWei,
        ),
      ),
    ],
  );
  print('UserOpHash: ${res.userOpHash}');

  print('Waiting for transaction...');
  final ev = await res.wait();
  print('Transaction hash: ${ev?.transactionHash}');
```

Reply:
Great! Thanks for catching that. Fixed.